src/modules/esl-media-query/core/conditions/media-query-containers.ts

```diff
@@ -0,0 +1,83 @@
+import {Observable} from '../../../esl-utils/abstract/observable';
+import {ALL, NOT_ALL} from './media-query-base';
+
+import type {IMediaQueryCondition} from './media-query-base';
+
+/**
+ * Abstract multiple media conditions container
+ * @author Alexey Stsefanovich (ala'n)
+ *
+ * Observe all child items. Dispatch changes when the whole condition result is changed
```

Comment:
'Observes'? Not sure, because somewhere with '-s', but somewhere not

Reply:
Moved to #357 discussion